lib/rules/web/admin_console/4.4/notification.xyaml

```diff
@@ -0,0 +1,52 @@
+click_notification_drawer:
+  elements:
+  - selector:
+      xpath: //button[contains(@aria-label,'Notification Drawer')]
+    op: click
+    timeout: 30
+open_notification_drawer:
```

Comment:
Normally we'd better not mix click & check operation in one action. Let me try to explain
Given these scenarios in feature 
```
    When I run the :open_notification_drawer web action
    Then the step should succeed
    When I run the :close_notification_drawer web action
    Then the step should succeed
```
At the first glance, we just notice that we're opening & close notification drawer, didn't know we also checked that `Notification` headings appear & disappear

Reply:
This time we can just leave them as they are